README.md

```diff
@@ -7,6 +7,10 @@ This script goes through all explicitly installed packages. For each package it
 ### Usage
 Have a look at the flags, since the default behaviour might not do all you want, and might not be most convenient for you.
 
+### Dependencies Required
+- aur/package-query
+- Yaourt (deprecated)
```

Comment:
Yaourt is not needed anymore (and neither is yay, I'm using pacman directly now)

You can remove this line.